### PR TITLE
Convert Login and Home page layout to Bootstrap

### PR DESCRIPTION
**Pull request by murex971**
### Description
implementation of Bootstrap

### CODE REVIEW

**Comment by shucon** [comment]
Please try to use better commit message and sign-off each commit.

File: templates/home/index.twig [hunk 1]
```diff
@@ -184,6 +185,6 @@
       </ul>
     </div>
   </div>
-</div>
+</div></div></div>
```
Comment (shucon) [inline templates/home/index.twig]: Proper indentation of closing tags, and each tag in new line is required.

File: templates/login/header.twig [hunk 2]
```diff
@@ -2,6 +2,8 @@
     <div id="modalOverlay">
 {% endif %}
 <div class="container{{ add_class }}">
+<div class="row">
+<div class="col-12">
```
Comment (shucon) [inline templates/login/header.twig]: Closing tags for them are not present.

**Review by shucon — CHANGES_REQUESTED** [review]

**Review by MauricioFauth — COMMENTED** [review]
Nice work!

You should try to keep the style of the themes (pmahomme, Metro and Original) as same as possible.
For example, the login page is looking different:
Screenshot from 2019-06-01 14-50-38

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Also, don't forget to add your signature tag in the commit messages with `git commit -s`.

**Comment by murex971** [comment]
> Nice work!
> 
> You should try to keep the style of the themes (pmahomme, Metro and Original) as same as possible.
> For example, the login page is looking different:
> Screenshot from 2019-06-01 14-50-38

I will be working on forms particularly once I am done with the grids part.

**Comment by mauriciofauth** [comment]
Closed in favor of #15302

**Pull request closed without merging**